services/director-v2/src/simcore_service_director_v2/modules/dask_client.py

```diff
@@ -195,9 +195,7 @@ def _comp_sidecar_fct(
         ) -> TaskOutputData:
             """This function is serialized by the Dask client and sent over to the Dask sidecar(s)
             Therefore, (screaming here) DO NOT MOVE THAT IMPORT ANYWHERE ELSE EVER!!"""
-            from simcore_service_dask_sidecar.tasks import (  # type: ignore
-                run_computational_sidecar,
-            )
+            from simcore_service_dask_sidecar.tasks import run_computational_sidecar
```

Comment:
maybe move this import at the top of the module?

Reply:
written just above is:
```"""This function is serialized by the Dask client and sent over to the Dask sidecar(s)
            Therefore, (screaming here) DO NOT MOVE THAT IMPORT ANYWHERE ELSE EVER!!"""```

Reply:
;)